week-1/interpret/to-pounds.js

```diff
@@ -2,18 +2,19 @@ const penceString = "399p";
 
 const penceStringWithoutTrailingP = penceString.substring(
   0,
-  penceString.length - 1
+  penceString.length - 1 // takes out the p with -1
 );
 
-const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0"); // padStarts add a zero infront of the number
```

Comment:
padStart will only add 0 in front in certain situations. What situations are that? In this case, how many 0s are added in front of the number?